docs/notebooks/introduction.rst

```diff
@@ -184,4 +189,41 @@ You can download a specific Notebook checkpoint from Neptune to Jupyter.
   - Checkpoint
 
 
-3. Click **Download**.
\ No newline at end of file
+3. Click **Download**.
+
+Using the Neptune CLI
+---------------------
+
+You can create Notebooks and update Notebook checkpoints in Neptune from the command line.
+
+Using CLI commands is an alternative if you prefer not to use the `neptune-notebooks extensions <installation.html>`_ in Jupyter or JupyterLab.
+
+.. note:: Syncing Notebook checkpoints using the neptune-notebooks extension is highly recommended!
+
+
+Create a new Notebook
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+    neptune notebook sync --project USERNAME/PROJECT_NAME your_notebook.ipynb --new
```

Comment:
`USERNAME` here is actually organization name when the project belongs to a team, instead of an individual. I don't know if we have a convention regarding this throughout the docs - maybe it's ok.

Reply:
Not sure, I think @moshekruger would be better suited to answer that.

Reply:
We'll fix it with @moshekruger